run_project.md

```diff
@@ -0,0 +1,44 @@
+# How to run the project after installing everything
+This is for **very beginners** like me 😊  
+
+Experienced members helped me build the environment and set everything up. Following are the steps after that. Please correct me if anything is wrong!
+
+## Tools
+- Docker: Run applications in an isolated environment
+- ChromaDB: The database to keep your data
```

Comment:
I feel these descriptions are not technically correct and misleading to some extent. Also, these components should be mentioned in README because it is what all the collaborators will see first. Please consider rephrase them or add them in README

Reply:
I will rephrase them and add them in README👍

Reply:
Actually, I will think over it more and decide what to do